core/src/test/java/hudson/FilePathTest.java

```diff
@@ -242,7 +242,7 @@ public void close() throws IOException {
                     throw x;
                 }
             } finally {
-                toF.chmod(700);
+                toF.chmod(0700);
```

Comment:
Not necessary but looked like a typo in the original test.

Reply:
Yup.